### ISSUE TITLE
OSX: Freezes when opening 3 files with same name

### ISSUE THREAD

**Issue by nojacko** [atom/atom#10161]
Found this issue when using Atom for an Ember CLI project. 

Atom Version 1.3.2

## Steps to reproduce

![screen shot 2015-12-22 at 15 50 48](https://cloud.githubusercontent.com/assets/1667141/11959163/d0940cea-a8c3-11e5-9a10-fe164c0c2f5a.png)

* The above folder structure. These command can create the structure: 
```
mkdir -p project/controllers/test/;
mkdir -p project/routes/test/;
mkdir -p project/controllers/test2;
touch project/controllers/test/index.js;
touch project/controllers/test2/index.js;
touch project/routes/test/index.js;
```
* Click controllers/test/index.js
* Click routes/test/index.js
* Click controllers/test2/index.js
* It should the freeze and Atom Helper process hits 100% CPU and never recovers.

**Comment by 50Wliu** [comment]
This has already been fixed in the 1.4 betas.

**Comment by nojacko** [comment]
:+1:
